nixos/modules/services/continuous-integration/github-runner/options.nix

```
@@ -170,4 +170,16 @@ with lib;
     default = null;
     defaultText = literalExpression "username";
   };
+
+  workDir = mkOption {
+    type = with types; nullOr path;
```

After testing, I realized that I would like to be able to use a [systemd specifier](https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Specifiers) here. Maybe it's in fact more flexible to use a string type here. Sorry about the back and forth.

```suggestion
    type = types.str;
```

---

I changed it to `type = with types; nullOr str;`, the default with `null` is different for `services.github-runner` vs `services.github-runners` and I'm not sure if that can be expressed here.

---

Ah right, yes!